connector.go

```diff
@@ -77,7 +77,7 @@ func NewConnector() (*Connector, error) {
 
 	//	Create kafka Produce channel and provide an outlet to connector object
 	eventSink := make(chan protoreflect.ProtoMessage)
-	go c.InitProduceChannel(eventSink)
+	go c.initProduceChannel(eventSink)
```

Comment:
I notice that the timestamp is not set when passing to `ProduceMsg()`

Reply:
it is in the `msg`
```c.ProducerInterface.ProduceMsg(topic, msg, nil, time.Time{}, nil)```

Reply:
But it's always `time.Time{}`

Reply:
That tells `ProduceMsg` to always use the timestamp in the `msg`.
If some other caller of `ProduceMsg` wants to set the timestamp themselves, they can send a non-zero timestamp to override the one in the `msg`.

Reply:
Inside the `c.initProduceChannel()`, it always passes `time.Time{}`, how would caller set the timestamp if they wanna use `connector` lib?

Reply:
They set the timestamp inside the protobuf message. If they want more customized solution, they can use kafkautils lib and `ProduceMsg` method and then they can set timestamp to an arbitrary value.